src/main/java/wellnus/focus/feature/Session.java

```diff
@@ -0,0 +1,79 @@
+package wellnus.focus.feature;
+
+
+import java.util.ArrayList;
+
+/**
+ * Represents a session of Countdown objects.
+ * A session is a sequence of Countdown objects represented by an ArrayList.
+ */
+public class Session {
```

Comment:
As per one of my comments, add in a method to reset the state of the timer to allow for consecutive Sessions to be made. Alternatively, destroy the session object and create a new instance.

Reply:
Will leave it to be implemented, current method of doing so is not foolproof.